File: ejercicios/c++/ejercicio075.cpp
#include <iostream>
using namespace std;

bool validador_recursivo(int m, int n){	if(m == 0)		return false;
	if(m%10 == n)
		return true;	else		return validador_recursivo(m/10,n);}

int main() {	int numero, digito;	cout << "ingrese un numero entero : ";	cin >> numero;
	cout << "ingrese el digito a verificar : ";
	cin >> digito;	if (validador_recursivo(numero, digito))		cout << "el digito " << digito << " se encuentra en " << numero << endl;	else		cout << "el digito " << digito << " no se encuentra en " << numero << endl;	return 0;}

File: ejercicios/python/ejercicio075.py
def validador_recursivo(m, n):
   if(m == 0):
      return False
   if(m%10 == n):
      return True
   else:
      return validador_recursivo(int(m/10),n);

if __name__ == "__main__":
   numero = int(input("ingrese un numero : "))
   digito = int(input("ingrese el digito a verificar : "))
   if (validador_recursivo(numero, digito)):
      print("el digito  " + str(digito) + " se encuentra en " + str(numero))
   else:
      print("el digito  " + str(digito) + " no se encuentra en " + str(numero))